### PR TITLE
Add QR Code

### PR DESCRIPTION
**Pull request by aureooms**
**https://github.com/aureooms/awesome-qr-code**

**This is a curated list of QR code libraries, software and resources.** I created it because I could not find any other such list. I have used some of the listed items and even created some.

I have reviewed [the following open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+reviewed-by%3Aaureooms): #1816, #1851, #1868, #1869.

## Requirements for your pull request

- [x] **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
- [x] You have read and understood the [instructions for creating a list](create-list.md).
- [x] This pull request has a title in the format `Add Name of List`.
- [x] Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** 
- [x] Your entry should be added at the bottom of the appropriate category.

## Requirements for your Awesome list

- [x] **Has been around for at least 30 days.**
- [x] Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues.
- [x] The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- [x] It's the result of hard work and the best I could possibly produce.
- [x] The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
- [x] The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
- [x] Non-generated Markdown file in a GitHub repo.
- [x] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics).
- [x] Not a duplicate. Please search for existing submissions.
- [ ] Only has awesome items. Awesome lists are curations of the best, not everything.
- [ ] Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- [x] Includes a project logo/illustration whenever possible.
	- [x] Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- [x] The image should link to the project website or any relevant website.
	- [x] **The image should be high-DPI.** Set it to maximum half the width of the original image.
- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- [x] Includes the [Awesome badge](awesome.md#awesome-badge).
	- [x] Should be placed on the right side of the readme heading.
	- [x] Should link back to this list.
- [x] Has a Table of Contents section.
	- [x] Should be named `Contents`, not `Table of Contents`.
	- [x] Should be the first section in the list.
	- [x] Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- [x] Has an appropriate license.
	- [x] Place a file named `license` or `LICENSE` in the repo root with the license text.
	- [x] **Do not** add the license name or text to the readme. GitHub already shows the license name at the top of the repo.
- [x] Has [contribution guidelines](awesome.md#include-contribution-guidelines).
	- [x] The file should be named `contributing.md`. Casing is up to you.
- [x] Has consistent formatting and proper spelling/grammar.
	- [x] The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- [x] The description starts with an uppercase character and ends with a period.
	- [x] Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- [x] Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- [x] Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- [x] Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

### CODE REVIEW

**Comment by aureooms** [comment]
unicorn

**Comment by n370** [comment]
Hey @aureooms I believe your `LICENSE` file isn't properly formatted. You'll want to take a look at https://github.com/sindresorhus/awesome/blob/main/license and https://choosealicense.com/licenses/cc0-1.0 to correct it.

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by aureooms** [comment]
Removed one instance of hard wrapping (https://github.com/aureooms/awesome-qr-code/commit/8d44f3df97c1dc796888eb02685ff6526f449cac).

**Comment by aureooms** [comment]
Used correct naming for *[Wi-Fi](https://en.wikipedia.org/wiki/Wi-Fi)* and *[QR code](https://en.wikipedia.org/wiki/QR_code)*: see https://github.com/aureooms/awesome-qr-code/commit/d21df2630c1c09651b7575578543f1b2b815a0ba and https://github.com/aureooms/awesome-qr-code/commit/93169c96d92ee1ae2c989f810a1f774c89de9e33.

**Comment by aureooms** [comment]
Added full license text (https://github.com/aureooms/awesome-qr-code/commit/8082719fcb8e105ac6a9238e8f1802e971c48cab).

**Comment by aureooms** [comment]
Added logo (https://github.com/aureooms/awesome-qr-code/commit/990dbf0abf07a1b5b36e8a4fb60241fe09287232).

**Comment by Jogai** [comment]
The list looks good to me.

**Comment by wernerkrauss** [comment]
Should the "Contributing" section be moved to CONTRIBUTING.md?

**Comment by j0hnm4r5** [comment]
This isn't really something worthy of stopping the merge, but the "Libraries" section in particular could use a little work on hierarchy.

The two sub-categories — "Readers" and "Generators" — are getting lost in the jumble of the language headers. Maybe pull them out into their own top-level categories?

File: readme.md [hunk 1]
```diff
@@ -707,6 +707,7 @@
 - [Biomedical Information Extraction](https://github.com/caufieldjh/awesome-bioie#readme) - How to extract information from unstructured biomedical data and text.
 - [Web Archiving](https://github.com/iipc/awesome-web-archiving#readme) - An effort to preserve the Web for future generations.
 - [WP-CLI](https://github.com/schlessera/awesome-wp-cli#readme) - Command-line interface for WordPress.
+- [QR codes](https://github.com/aureooms/awesome-qr-code#readme) - QR code is the trademark for a type of matrix barcode.
```
Comment (sindresorhus) [inline readme.md:710]: The description could be improved. The fact that it's a trademark is completely uninteresting in this context. I think it should also better explain exactly what a QR code is.

Comment (sindresorhus) [reply readme.md:710]: Also, title-case on the title.

Comment (aureooms) [reply readme.md]: Thanks! I just tried to fix that in 4f4efca and 7c17b10.

**Comment by sindresorhus** [comment]
You need to fix the merge conflict.

**Comment by aureooms** [comment]
> You need to fix the merge conflict.

Done!

File: readme.md [hunk 2]
```diff
@@ -726,6 +726,7 @@
 - [Credit Modeling](https://github.com/mourarthur/awesome-credit-modeling#readme) - Methods for classifying credit applicants into risk classes.
 - [Ansible](https://github.com/KeyboardInterrupt/awesome-ansible#readme) - A Python-based, open source IT configuration management and automation platform.
 - [Biological Visualizations](https://github.com/keller-mark/awesome-biological-visualizations#readme) - Interactive visualization of biological data on the web.
+- [QR Code](https://github.com/aureooms/awesome-qr-code#readme) - A QR code is a type of matrix barcode that can be used to store and share up to 2953 bytes of arbitrary information.
```
Comment (sindresorhus) [inline readme.md:729]:
```suggestion
- [QR Code](https://github.com/aureooms/awesome-qr-code#readme) - A type of matrix barcode that can be used to store and share a small amount of information.
```

**Comment by sindresorhus** [comment]
You need to fix the merge conflict.

**Comment by tahayigitmelek** [comment]
[This](https://chrome.google.com/webstore/detail/qr-code-generator/afpbjjgbdimpioenaedcjgkaigggcdpp?hl=en)  Google Chrome Extension is really useful.

**Comment by Jogai** [comment]
> [This](https://chrome.google.com/webstore/detail/qr-code-generator/afpbjjgbdimpioenaedcjgkaigggcdpp?hl=en) Google Chrome Extension is really useful.

Its built in nowadays...

**Comment by gianni-dalerta** [comment]
Was looking for lists to review and this was actually a list I needed for a project. 💯

**Comment by aureooms** [comment]
> This isn't really something worthy of stopping the merge, but the "Libraries" section in particular could use a little work on hierarchy.
> 
> The two sub-categories — "Readers" and "Generators" — are getting lost in the jumble of the language headers. Maybe pull them out into their own top-level categories?

I decided to remove these sub-categories and instead add "tags" to the entries, see https://github.com/aureooms/awesome-qr-code/commit/2da88767f833ea7c148edfe9fe82fc22acac71ff. What do you think?

**Comment by aureooms** [comment]
> Should the "Contributing" section be moved to CONTRIBUTING.md?

I moved most of the stuff to `CONTRIBUTING.md` and kept a minimal pointer in `README.md` (https://github.com/aureooms/awesome-qr-code/commit/0ef6cd8a158b3cc385507ee0a3c93ddf60449219).

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1365249090196443138

**Pull request merged**